### DISCUSSION TITLE
gh webhook forward failing on GitHub Enterprise

### DISCUSSION THREAD

**Discussion by frossplaid** [community/community#51865] [Enterprise]
### Select Topic Area

Question

### Body

On our GitHub Enterprise installation, I created a repo that I am an admin on, then tried to use the GH CLI to set up webhook forwarding to my local machine:

```
% gh webhook forward -H github.my-org.com --repo="fross/my-repo-name" --events="pull_request" --url="http://localhost:8081/github-webhooks"
Error: error creating webhook: HTTP 422: Validation Failed (https://github.my-org.com/api/v3/repos/fross/my-repo-name/hooks)
Hook.name is invalid
```

Is there something obvious I'm missing here, or a configuration setting we should check? Or is this a bug in the CLI extension?

**Comment by alleer22** [comment]
Hey, do you have a solution?

**Comment by skedwards88** [comment]
The extension will only work on free/pro/team and GHEC accounts, based on the versions that the corresponding documentation is available on: https://docs.github.com/en/webhooks/testing-and-troubleshooting-webhooks/receiving-webhooks-with-the-github-cli

<img width="961" alt="screenshot of version menu on docs" src="https://github.com/community/community/assets/25328854/3a9fe177-1c3c-44b7-ae76-4069ff63c63b">

**Comment by collinmurd** [comment]
As of GHES 3.12, the webhook CLI still isn't supported. For anyone looking for a solution without setting up a proxy server, I wrote a [`gh` extension  that's all API driven](https://github.com/collinmurd/gh-ghes-webhook). It does not require the websocket connection that `gh webhook` uses.
